src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -19,6 +19,7 @@
 import io.openliberty.tools.intellij.it.fixtures.DialogFixture;
 import io.openliberty.tools.intellij.it.fixtures.ProjectFrameFixture;
 import io.openliberty.tools.intellij.it.fixtures.WelcomeFrameFixture;
+import io.openliberty.tools.intellij.util.LocalizedResourceUtil;
```

Comment:
this import is not being used, please remove

Reply:
@TrevCraw removed import.